### PR TITLE
simplify code using object shorthand in storage views

### PR DESCRIPTION
**Pull request by Damans227**
### Description

- This PR is related to the issue: `ui: Simplify code` #4956
- It uses [object shorthand notation from Javascript - ES6 ](https://alligator.io/js/object-property-shorthand-es6/)to simplify the `.vue` files in storage view. 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by Damans227** [comment]
@davidjumani @utchoang  Ptal, thx!

**Comment by utchoang** [comment]
@davidjumani In my opinion, the shorthand object does not apply to the data() object, but only to the objects that are used in the handling functions of the method, mounted, created,...? What do you think?

**Comment by sureshanaparti** [comment]
@blueorangutan ui

File: ui/src/views/storage/AttachVolume.vue [hunk 1]
```diff
@@ -59,9 +59,11 @@ export default {
     }
   },
   data () {
+    let virtualmachines = [];
```
Comment (utchoang) [inline ui/src/views/storage/AttachVolume.vue:62]: Please remove the ";" character

**Review by utchoang — CHANGES_REQUESTED** [review]

File: ui/src/views/storage/AttachVolume.vue (hunk 1 above)
Comment (utchoang) [reply ui/src/views/storage/AttachVolume.vue:62]: Apply for the same changes

**Comment by davidjumani** [comment]
While using object shorthand is a good idea, creating variables with empty / null / static values just to be returned a couple of lines down can actually reduce the readability. @Damans227 Can you look at other areas where doing so will make reading the code easier or simplify it ? Thanks

**Comment by Damans227** [comment]
> While using object shorthand is a good idea, creating variables with empty / null / static values just to be returned a couple of lines down can actually reduce the readability. @Damans227 Can you look at other areas where doing so will make reading the code easier or simplify it ? Thanks

That makes sense. I will go ahead and close this PR.

**Pull request closed without merging**